manifest

```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
+	pipewire-pulse \
+	pipewire-jack \
+	gst-plugin-pipewire \
```

Comment:
What is this for?

Reply:
The gst-plugin is needed for streaming/recording gamescope session, gnome-shell or any other pipewire stream. It only adds 46Kb to the compressed image and 127Kb fully installed